plugins/python/src/test/kotlin/io/paddle/installTask/InstallTaskContainerTest.kt

```diff
@@ -0,0 +1,78 @@
+package io.paddle.installTask
+
+import io.paddle.plugin.python.dependencies.interpreter.InterpreterVersion
+import io.paddle.plugin.python.extensions.globalInterpreter
+import io.paddle.project.PaddleProjectProvider
+import io.paddle.testExecutor.AbstractTestContainerTest
+import org.junit.jupiter.api.*
+import org.testcontainers.junit.jupiter.Testcontainers
+import java.io.File
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+@Testcontainers
+class InstallTaskContainerTest :
+    AbstractTestContainerTest("python:3.10-buster" /* TODO: change me. A container should have c/cpp compiler & make tools to build python */) {
+    private lateinit var rootDir: File
+
+    @AfterEach
+    fun cleanupResources() {
+        rootDir.resolve(".venv").deleteRecursively()
+        rootDir.resolve(".paddle").deleteRecursively()
+    }
+
+    @Test
+    fun `minimal project resolving`() {
+        rootDir = resources.resolve("minimal-project")
+        val projectProvider = PaddleProjectProvider.getInstance(rootDir)
```

Comment:
Let's add a method like `PaddleProject.create(rootDir: File)` to deal with such copy-pastes later